src/ci/docker/x86_64-gnu-tools/publish.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution and at
+# http://rust-lang.org/COPYRIGHT.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import sys
+import re
+import json
+import copy
+import datetime
+import collections
+
+# List of people to ping when the status of a tool changed.
+MAINTAINERS = {
+    'miri': '@oli-obk @RalfJung @eddyb',
+    'clippy-driver': '@Manishearth @llogiq @mcarton @oli-obk',
+    'rls': '@nrc',
+    'rustfmt': '@nrc',
+}
```

Comment:
Having this list somewhere in a script deep in the `ci/docker` folder may not be the best place. Is there a precedent for where to put such "repository configuration files"?

Reply:
@RalfJung This script can be placed elsewhere. Maybe `src/tools/publish_toolstate.py`?

Reply:
That would certainly help. I was originally thinking of separating configuration and code, but given the size of the script, maybe that's overdoing it.